gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -168,3 +187,13 @@ def title(event):
         f"[{ALERT_DETAILS[log]['DOC_TITLE']}] externally visible to [{sharing_scope}] with "
         f"[{alert_access_scope}] access"
     )
+
+
+def severity(event):
+    log = event.get("p_row_id")
+    if ALERT_DETAILS[log]["TARGET_USER_EMAILS"] != ["<UNKNOWN_USER>"]:
+        for address in ALERT_DETAILS[log]["TARGET_USER_EMAILS"]:
+            domain = address.split("@")[1]
```

Comment:
are there any cases where this might not contain an "@"? (some malformed email of sorts?)

Reply:
I don't _think_ so.  I would expect the email address should be validated by Gdrive before it allows a share.